<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MGRSGridForm</class>
 <widget class="QWidget" name="MGRSGridForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>44</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>MGRS:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>100 km</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 km</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 km</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100 m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 m</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_coosys"/>
   </item>
   <item>
    <widget class="QPushButton" name="searchButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="mgrs.qrc">
       <normaloff>:/Resources/search.png</normaloff>:/Resources/search.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mgrs.qrc"/>
 </resources>
 <connections/>
</ui>
